platform/polkadot/formatter.go

```diff
@@ -0,0 +1,21 @@
+package polkadot
+
+import (
+	"math"
+	"math/big"
+
+	"github.com/trustwallet/blockatlas/pkg/blockatlas"
+)
+
+func ParseAmount(s string, decimals uint) blockatlas.Amount {
```

Comment:
What does this do? We already have decoders for this

If it’s unique - we should have tests

Reply:
tests added